casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -103,6 +103,13 @@ class RhoSpec(
           3,
           105L,
           ctx => DeployerIdContract.get(ctx)(_, _)
+        ),
+        SystemProcess.Definition[F](
+          "rho:test:crypto:secp256k1Sign",
+          Runtime.byteName(106),
```

Comment:
please double check there's no conflict on that magic number. Ideally, we'd have all those magic numbers extracted as constants in some object, so it's less easy to shoot oneself in the foot.

Reply:
Done; I had this discussion already with @odeac and thought I created a ticket for this already. But can't find it anymore and therefore creating a new one: https://rchain.atlassian.net/browse/RCHAIN-3801